upnp_port_forward/client.py

```diff
@@ -39,9 +47,19 @@ def setup_port_map(
     if not devices:
         raise PortMapFailed("No UPnP devices available")
 
+    if wan_service_name:
+        if not wan_service_name.startswith("WAN"):
```

Comment:
Can we actually know that all of the WAN services will actually have a name starting with `"WAN"`?

Reply:
I just saw too much emphasis on the 'WAN' in: `--upnp-service-name WANSomethingOrOther`.
But anyway it was more a way to start a discussion on possible checks on the user input.
I'll remove it if we don't need any checks. But do we need this feature at all ? (see my comment here https://github.com/ethereum/trinity/pull/2116#issuecomment-737380888 )

Reply:
I think we still want this feature (but without the validation). That makes the library usable even if their device isn't yet supported